css/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <div>This is a Landing Page</div> 
+      <div><img id="header-img" src="Images/IMG_5372.JPG" alt="Photo of a puppy"/ width="300px"></div>
+      
+      <div><nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#header">Index</a></li>
+          <li><a class="nav-link" href="#video">Video</a></li>
+          <li><a class="nav-link" href="#header">Contact</a></li>
+        </ul>
+      </nav></div>
+
+    </header>
+    
+    <div>
```

Comment:
It is a better code practice to use semantic tags as "section" instead of "div"...sorry for be so detailed with you!